src/me/ryanhamshire/GriefPrevention/events/BlockEventHandler.java

```diff
@@ -305,26 +305,20 @@ public void onBlockPlace(ChangeBlockEvent.Place event) {
                 transaction.setValid(false);
                 continue;
             }
-    
+
             // if the block is being placed within or under an existing claim
             PlayerData playerData = this.dataStore.getPlayerData(player.getWorld(), player.getUniqueId());
             Claim claim = this.dataStore.getClaimAt(block.getLocation().get(), true, playerData.lastClaim);
 
             if (claim != null) {
                 playerData.lastClaim = claim;
-    
-                // warn about TNT not destroying claimed blocks
-                if (block.getState().getType().equals(BlockTypes.TNT) && !claim.areExplosivesAllowed && playerData.siegeData == null) {
```

Comment:
My IDE automatically fixed some of the formatting - these lines are some of the real changes.